scout/demo/custom_evaluation_terms_example.json

```diff
@@ -0,0 +1,270 @@
+[

```

Comment:
This file contains the default evaluation terms. It can be modified to include all the terms used in Lund